bundles/org.openhab.binding.kostalinverter/src/main/resources/ESH-INF/config/SecondGeneration.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0" xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+	<config-description uri="thing-type:kostalpiko1020_config">
+		<parameter name="url" type="text">
+			<label>Url</label>
+			<description>URL to inverter.</description>
+			<required>true</required>
```

Comment:
The `required` tag is deprecated. You can add it as an attribute to the parameter tag.